### ISSUE TITLE
Shipwright Build Operator does not have permission to delete cluster roles

### ISSUE THREAD

**Issue by adambkaplan** [shipwright-io/operator#71]
## Expected Behavior

Deleting a `ShipwrightBuild` object deletes the Build controller deployment and any other objects deployed by the operator.

## Actual Behavior

`ShipwrightBuild` is never deleted - operator does not have sufficient permissions to delete the new default cluster roles:

```
1.651247128306714e+09 ERROR controllers.ShipwrightBuild deleting manifest's resources {"namespace": "", "name": "shipwright-build", "targetNamespace": "shipwright-build", "error": "clusterroles.rbac.authorization.k8s.io \"shipwright-build-aggregate-view\" is forbidden: User \"system:serviceaccount:openshift-operators:shipwright-operator\" cannot delete resource \"clusterroles\" in API group \"rbac.authorization.k8s.io\" at the cluster scope"}
```

**Comment by qu1queee** [comment]
From grooming: Should we consider testing both install/removal of the controllers? @adambkaplan

**Comment by adambkaplan** [comment]
Yes - we need to add a proper e2e test suite that does the following:

1. Create a `ShipwrightBuild` object.
2. Verify that the build controller was deployed.
3. Verify the user-facing roles are deployed.
4. Deletes the `ShipwrightBuild` object
5. Verifies the build controller is removed
6. Verifies the user-facing roles are removed

**Comment by SaschaSchwarze0** [comment]
From grooming: we will close this isue with above PR, a separate issue will be opened to cover the missing e2e test case.
